interfaces/cython/cantera/ctml_writer.py

```diff
@@ -1627,11 +1662,13 @@ def __init__(self,
                  kf = None,
                  id = '',
                  order = '',
+                 rateCoeff = '',
```

Comment:
I agree - I've never really liked the tag `rateCoeff`...nor the option `exchangecurrentdensity`, which one can use to specify the rate coefficient type.

1. I agree that `rate_coeff_type` is the way to go for this label.
2. For the latter, I would rather use `butler_volmer`, both because it's shorter, but also because the exchange current density can be used in multiple ways.  For the Butler-Volmer implementation, the charge-transfer current would be:
<a href="https://www.codecogs.com/eqnedit.php?latex=i&space;=&space;i_\circ\left[\exp\left(\frac{\beta&space;n_{\rm&space;elec}F&space;\eta}{RT}&space;\right)-\exp\left(\frac{-(1-\beta)&space;n_{\rm&space;elec}F&space;\eta}{RT}&space;\right)\right&space;]" target="_blank"><img src="https://latex.codecogs.com/gif.latex?i&space;=&space;i_\circ\left[\exp\left(\frac{\beta&space;n_{\rm&space;elec}F&space;\eta}{RT}&space;\right)-\exp\left(\frac{-(1-\beta)&space;n_{\rm&space;elec}F&space;\eta}{RT}&space;\right)\right&space;]" title="i = i_\circ\left[\exp\left(\frac{\beta n_{\rm elec}F \eta}{RT} \right)-\exp\left(\frac{-(1-\beta) n_{\rm elec}F \eta}{RT} \right)\right ]" /></a>
Another implementation would be the Tafel equation, where at large enough overpotentials the 2nd term in Butler-Volmer becomes negligible, and is dropped:
<a href="https://www.codecogs.com/eqnedit.php?latex=i&space;=&space;i_\circ\exp\left(\frac{\beta&space;n_{\rm&space;elec}F&space;\eta}{RT}&space;\right)" target="_blank"><img src="https://latex.codecogs.com/gif.latex?i&space;=&space;i_\circ\exp\left(\frac{\beta&space;n_{\rm&space;elec}F&space;\eta}{RT}&space;\right)" title="i = i_\circ\exp\left(\frac{\beta n_{\rm elec}F \eta}{RT} \right)" /></a>
Anyway, I think both would be valuable to enable, and the simplest would be to have the specified at the `rate_coeff_type` level.